src/Features/CSharp/GenerateMember/GenerateVariable/CSharpGenerateVariableService.cs

```diff
@@ -121,6 +121,11 @@ protected override bool ContainingTypesOrSelfHasUnsafeKeyword(INamedTypeSymbol c
                 return false;
             }
 
+            if (expression.IsParentKind(SyntaxKind.ConditionalAccessExpression))
+            {
+                return true;
+            }
+
```

Comment:
We now explicitly check since we know that the generate variable state machine can handle this case and bypass CanReplaceWithLValue entirely.